pxr/usd/usd/CMakeLists.txt

```diff
@@ -299,6 +299,13 @@ pxr_build_test_shared_lib(TestUsdResolverChangedResolver
         testenv/TestUsdResolverChangedResolver.cpp
 )
 
+pxr_build_test(testCoreUsdValidators
```

Comment:
Also this makes me realize I should have included my CompositionErrorValidator test in something like this, instead of bundling it in the validationRegistry test. I will do move it internally. But thanks!